cirq-core/cirq/sim/simulation_state.py

```diff
@@ -187,7 +186,11 @@ def with_qubits(self: TSelf, qubits) -> TSelf:
         Regurns:
             A new subclass object containing the extended state space.
         """
+        # FIXME: SimulationState.__init__ requires the `state` argument, but
```

Comment:
Could you open an issue for this and reference it here?

Reply:
Done - opened #5721 and added a TODO link to that issue.